graphix/sim/statevec.py

```diff
@@ -264,11 +268,27 @@ def ptrace(self, qargs):
         evals, evecs = np.linalg.eig(rho)  # back to statevector
         self.psi = np.reshape(evecs[:, np.argmax(evals)], (2,) * nqubit_after)
 
+    def truncate_one_qubit(self, qarg):
```

Comment:
is `truncate` a good word here? let me have a think for a bit

Reply:
truncate sounds like removing the last qubit?

Reply:
Hmm... `discard` or `remove` ?